docs/integrations/rsyslog/index.rst

```diff
@@ -0,0 +1,226 @@
+   certificate)
+
+-  **key** - (PEM format) client key if the server requires client
+   authentication
+
+-  **cert** - (PEM format) client cert to use
+
+Add rsyslog integration to service
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This can be configured in the Aiven Console by navigating to the target
+service overview page and then scrolling down to the Manage Integrations
+button.
+
+
+.. image:: /images/integrations/remote-syslog-integration_image2.png
+
+You should be able to select your previously configured Rsyslog service
+integration by clicking Use integration in the modal window.
+
+.. image:: /images/integrations/remote-syslog-integration_image3.png
+
+Alternately, with the Aiven Client, first you need the id of the
+endpoint previously created
+
+::
+
+   avn service integration-endpoint-list --project your-project
+   ENDPOINT_ID                           ENDPOINT_NAME   ENDPOINT_TYPE
+   ====================================  ==============  =============
+   618fb764-5832-4636-ba26-0d9857222cfd  example-syslog  rsyslog
+
+Finally you can link the service to the endpoint
+
+::
+
+   avn service integration-create --project your-project \
+       -t rsyslog -s your-service \
+       -D 618fb764-5832-4636-ba26-0d9857222cfd
+
+Integrating with a third party syslog service
+---------------------------------------------
+
+All integrations can be configured using the Aiven Console or the Aiven
+CLI though the examples are easier to copy and paste in the CLI form.
+
+papertrail
+~~~~~~~~~~
+
+As papertrail identifies the client based on the server and port  you
+only need to copy the appropriate values from the "Log Destinations"
+page and use those as the values for server and port respectively. You
+**do not need** the ca-bundle as papertrail's servers use certificates
+signed by known CAs. You also need to set the format to ``rfc3164`` .
+
+::
+
+   avn service integration-endpoint-create --project your-project \
+       -d papertrail -t rsyslog \
+       -c server=logsN.papertrailapp.com -c port=XXXXX \
+       -c format=rfc3164 -c tls=true
+
+loggly
+~~~~~~
+
+In addition to the server and port you also need a *customer token*
+which you then **need** to give as part of the ``-c sd`` parameter when
+creating the endpoint.
+
+::
+
+   avn service integration-endpoint-create --project your-project \
+       -d loggly -t rsyslog \
+       -c server=logs-01.loggly.com -c port=514 \
+       -c format=rfc5424 -c tls=true \
+       -c sd='TOKEN@NNNNN TAG="tag-of-your-choice"'
+
+sumo logic
+~~~~~~~~~~
+
+You need to the give the collector token as the ``-c sd`` parameter and
+use the server and port of the collector.
+
+::
+
+   avn service integration-endpoint-create --project your-project \
+       -d loggly -t rsyslog \
+       -c server=syslog.collection.XX.sumologic.com \
+       -c port=6514 \
+       -c format=rfc5424 -c tls=true \
+       -c sd='collector-token-string@NNNNN'
+
+Datadog
```

Comment:
I believe this should be part of the datadog section instead of embedded here and here maybe a link to there, what do you think?

Reply:
I kept this section as it is using `avn` cli to add the service integration but added a link to the datadog integration
```:doc:`Further details on send metrics and logs to Datadog </docs/integrations/datadog/index>```
Let me know if you have better suggestions.